.gitignore

```diff
@@ -1,4 +1,5 @@
 prom-label-proxy
 
 .idea
-.envrc
\ No newline at end of file
+.envrc
+.vscode
```

Comment:
could you remove this change?
self-note: we should remove .envrc and .idea too.

Reply:
Thank you for the feedback. I removed all of them(.envrc, .idea and .vscode)